app.ts

```diff
@@ -809,15 +808,15 @@ app.get(
 	isAuthenticated,
 	async function (req, res) {
 		const sensorTypes = configManager.sensorTypes
-		const sensorGroups = configManager.namedScales
+		const sensorScalesGroups = configManager.namedScales
 
 		const scales: SensorTypeScale[] = []
 
-		for (const [key, group] of sensorGroups) {
+		for (const [key, group] of sensorScalesGroups) {
 			for (const [, scale] of group) {
 				scales.push({
 					key: key,
-					sensor: group.name,
+					sensor: group.name || key,
 					unit: scale.unit,
 					label: scale.label,
 					description: scale.description,
```

Comment:
And I still don't understand for what you need `group: true`. I don't see where it is used.

Reply:
I was using it previously but not anymore, removing it now